backend/api/views/main.py

```diff
@@ -74,6 +53,34 @@ def create_person():
     )
 
 
+'''
+function that is called when you visit /response_rates
+Parameters
+    date: date string with the format MMDDYYYY
+    year: year string with format YY
+    optional tract_id: 11-digit tract id string
+    optional state: two digit id string
+Either date or year is required.
+'''
+@main.route("/response_rates", methods=["GET"])
+def get_response_rates():
+    responses_rate = None
+    
+    tract_id = request.args.get("tract_id", None)
+    date = request.args.get("date", None)
+    year = request.args.get("year", None)
+    state = request.args.get("state", None)
+
+    if date:
+        response_rates = get_response_rates_by_date(date, tract_id, state)
+    elif year:
+        response_rates = get_response_rates_by_year(year, tract_id, state)
+    else:
+        return create_response(status=442, message="Missing request parameters")
```

Comment:
is this a valid response code or do you mean 422

Reply:
lol i meant 422